docs/source/using/host-folders.adoc

```diff
@@ -0,0 +1,225 @@
+[[host-folders]]
+= Host Folders
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 2
+
+toc::[]
+
+[[overview]]
+== Overview
+
+Host folders are directories on the host which are shared between the host and VM.
+Host folders allow two way file synchronization between host and VM.
+The following sections discuss the various types of host folders, driver provided host folders, as well as the Minishift `hostfolder` command.
+
+[[driver-host-folders]]
+== Driver provided host folders
+
+Some drivers mount a default host folder into the VM in order to share files between the VM and the host.
+These folders are currently not configurable and differ for each driver and OS.
+
+.Driver provided host folders
+[cols=",,,",options="header",]
+|========================================
+|Driver |OS |HostFolder |VM
+|Virtualbox |Linux |/home |/hosthome
+|Virtualbox |OSX |/Users |/Users
+|Virtualbox |Windows |C://Users |/c/Users
+|VMWare Fusion |OSX |/Users |/Users
+|Xhyve |OSX |/Users |/Users
+|========================================
+
+NOTE: Host folder sharing is not implemented in the KVM and Hyper-V drivers.
+If you use one of these drivers, you need to use <<mounting-custom-shared-folders, Minishift specific host folder options>>.
```

Comment:
? 'mounting-custom-shared-folders' is a reference to a section, this is not a rendered text